### DISCUSSION TITLE
Is there a way to add a background to the marker?

### DISCUSSION THREAD

**Discussion by paolotiu** [tradingview/lightweight-charts#1319] [Q&A]
First of all, thank you so much for this library. It has made charting so much easier to work with. 

One feature I would like to know is possible though is a series marker could have a background color like the picture below?
<img width="256" alt="image" src="https://user-images.githubusercontent.com/61054234/229099053-e716a889-3f8a-4cdd-adc0-827c6c78f5bb.png">

Again, thank you so much!

**Comment by AgarwalPragy** [comment]
This will be really helpful. Markers often overlap with the candlestick values and become unreadable.
